README.md

```diff
@@ -35,6 +35,23 @@ The current Hilla support has some known limitations:
 Get started with `quarkus-hilla` by following the [Quick Start Guide](../../wiki/QuickStart)
 or download the [starter project](https://github.com/mcollovati/quarkus-hilla-starter).
 
+```xml
+<dependency>
+  <groupId>com.github.mcollovati</groupId>
+  <artifactId>quarkus-hilla</artifactId>
+  <version>2.0.0-alpha1</version>
+</dependency>
+```
+
+## Compatibility Matrix
+
+|                                                                                                  Quarkus-Hilla                                                                                                   |                                               Quarkus                                                |                                              Hilla                                               |                                              Vaadin                                              |
+|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:----------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------:|
+|  [![Maven Central](https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&versionPrefix=2)](https://central.sonatype.com/artifact/com.github.mcollovati/quarkus-hilla)   | ![Static Badge](https://img.shields.io/badge/QUARKUS-v3.1%2B-blue?style=for-the-badge&logo=Quarkus)  | ![Static Badge](https://img.shields.io/badge/HILLA-v2.1%2B-blue?style=for-the-badge&logo=Vaadin) | ![Static Badge](https://img.shields.io/badge/FLOW-v24.1%2B-blue?style=for-the-badge&logo=Vaadin) |
+| [![Maven Central](https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&versionPrefix=1)](https://central.sonatype.com/artifact/com.github.mcollovati/quarkus-hillchio) | ![Static Badge](https://img.shields.io/badge/QUARKUS-v2.16%2B-blue?style=for-the-badge&logo=Quarkus) | ![Static Badge](https://img.shields.io/badge/HILLA-v1.3%2B-blue?style=for-the-badge&logo=Vaadin) | ![Static Badge](https://img.shields.io/badge/FLOW-v23.3%2B-blue?style=for-the-badge&logo=Vaadin) |
```

Comment:
It would be good to replace `[Static Badge]` with the correct alternative text, e.g. for Hilla badge `[Hilla 2.1+]`